src/sagemaker_sklearn_extension/feature_extraction/sequences.py

```diff
@@ -101,17 +116,30 @@ def __init__(
         self.augment = augment
         self.interpolation_method = interpolation_method
         self.extraction_type = extraction_type
+        self.extraction_seed = extraction_seed
+        self.sequences_lengths_q25 = sequences_lengths_q25
 
     def fit(self, X, y=None):
         X = check_array(X, dtype=None, force_all_finite="allow-nan")
+
+        if self.sequences_lengths_q25 is None:
```

Comment:
Why not calculate if None?

Reply:
I'd like to simplify expected behavior here. If no provided sequences_lengths_q25, then we caller does not care about expansion, then we just include large enough default value.